browsertests/go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/google/pprof/browsertests
+
+go 1.19
+
+require (
+	github.com/chromedp/chromedp v0.9.2
+	github.com/google/pprof v0.0.0-20240521024322-9665fa269a30
```

Comment:
you'll want this to exercise the pprof code at HEAD, not this locked version

to do that, point this module at the local pprof module with something like this:

```
replace github.com/google/pprof => ../
```

and probably switch the require directive to `github.com/google/pprof v0.0.0` to make it clear this is not pinning to a specific pprof version